minimal/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "build": "react-router build",
+    "dev": "react-router dev",
+    "start": "react-router-serve ./build/server/index.js",
```

Comment:
Because this is not marketed as production ready and the README doesn't even mention `build`/`start` I removed the `NODE_ENV` stuff here and the `cross-env` dependency.  We can add it back in through if folks feel strongly

Reply:
No longer necessary anyway since you fixed that up